src/__experimental__/components/fieldset/fieldset.stories.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import notes from './documentation/notes.md';
```

Comment:
Could we create an `index.js` file and export notes there? It will keep documentation convention clean